data/json/items/tool/tailoring.json

```diff
@@ -85,7 +85,7 @@
     "use_action": {
       "type": "repair_item",
       "item_action_type": "repair_fabric",
-      "materials": [ "cotton", "leather", "lycra", "nylon", "wool", "fur", "faux_fur", "nomex", "kevlar", "kevlar_layered", "gutskin" ],
+      "materials": [ "cotton", "clay", "leather", "lycra", "nylon", "wool", "fur", "faux_fur", "nomex", "kevlar", "kevlar_layered", "gutskin" ],
```

Comment:
I don't think you can stitch clay together with a needle🤔

Reply:
Hi, the intent was so that you could engrave on clay with a needle. I thought about using scrap metal + lumps of clay while forging it so you could make a plaque for it, but I didn't see clay urns with metal plaques for sale online, plus joining them would be messy, you'd have to mold the metal to the shape of the urn and superglue it on or something, and it still wouldn't look great. So the options are to write on the urn with a pencil/marker, or to engrave it. If that change in tailoring.json also makes it possible to repair the urn...well that needs to get looked at again.

Reply:
This isn't engraving, this is repairing.

Reply:
Yeah, this action covers repairing rather than engraving. Knives are much more common than tailoring tools anyway, so that's kind of unnecessary.